elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -308,14 +289,17 @@ export class DelegatesList {
 		);
 
 		let usernames = { registeredDelegates: [] } as ChainStateUsernames;
+
 		if (usernamesBuffer) {
-			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-			const parsedUsername = JSON.parse(usernamesBuffer.toString('utf8'));
+			const parsedUsernames = codec.decode(
```

Comment:
```suggestion
			const parsedUsernames = codec.decode< DecodedUsernames>(
```

Reply:
Fixed locally to made the fix in your next comment below. Thanks for this suggestion.